PhysicsTools/PatAlgos/python/slimming/prunedGenParticles_cfi.py

```diff
@@ -20,12 +20,13 @@
 	"keep (4 <= abs(pdgId) <= 5)",
 # keep light-flavour quarks and gluons for parton-based jet flavour
 	"keep (1 <= abs(pdgId) <= 3 || pdgId = 21) & (status = 2 || status = 11 || status = 71 || status = 72) && pt>5", 
-# keep b and c hadrons for hadron-based jet flavour
-	"keep (400 < abs(pdgId) < 600) || (4000 < abs(pdgId) < 6000)",
-# keep onia states
-        "keep abs(pdgId) == 333",
-        "keep abs(pdgId) == 443 || abs(pdgId) == 100443 || abs(pdgId) == 10441 || abs(pdgId) == 20443 || abs(pdgId) == 445 || abs(pdgId) == 30443",
-        "keep abs(pdgId) == 553 || abs(pdgId) == 100553 || abs(pdgId) == 200553 || abs(pdgId) == 10551 || abs(pdgId) == 20553 || abs(pdgId) == 555",
+# keep b and c hadrons for hadron-based jet flavour, and decays for BPH studies
+	"keep++ (400 < abs(pdgId) < 600) || (4000 < abs(pdgId) < 6000)",
```

Comment:
didn't we agree to keep only first generation daughters? (given that packed Gen are anyhow available?)
How much the size increase changes if you change the above line to keep+ rather than keep++ ?